dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassDumperTest.java

```diff
@@ -43,7 +43,7 @@
 import org.junit.Test;
 
 public class ClassDumperTest {
-  private static final String EXAMPLE_JAR_FILE =
+  static final String EXAMPLE_JAR_FILE =
```

Comment:
why can't this be private?

Reply:
ClassReferenceGraphTest also uses it.

Renamed it to GRPC_CLOUD_FIRESTORE_JAR to be more specific.